Content.Shared/Implants/Components/ImplanterComponent.cs

```diff
@@ -82,6 +83,30 @@ public sealed partial class ImplanterComponent : Component
     [DataField(required: true)]
     public ItemSlot ImplanterSlot = new();
 
+    /// <summary>
+    /// If true, the implanter may be used to remove all kinds of (deimplantable) implants without selecting any.
+    /// </summary>
+    [DataField]
+    public bool AllowDeimplantAll = false;
+
+    /// <summary>
+    /// The subdermal implants that may be removed via this implanter
+    /// </summary>
+    [DataField]
+    public List<EntProtoId> DeimplantWhitelist = new();
+
```

Comment:
Prototype IDs shouldn't be used for whitelisting. The issue is it breaks inheritance, e.,g. if someone wants RedTrombone instead of just Trombone then the functionality breaks. It should use tags or components instead.

Reply:
I decided to continue with this implementation for the following reason:

A child implant can have two desirable behaviors. Either the child implant is available in the draw UI as its own implant (i.e. it lists both "trombone implant" and "red trombone implant", and you have to pick the correct one) *or* the child implant should be drawn based on its parent implant (i.e. if you select "trombone implant", you can also draw the red trombone one). 

To make sure both functionalities are available, I've added `DrawableProtoIdOverride` as a property to `SubdermalImplantComponent`. By default, child implants do not appear in the draw menu and thus are not drawable. From what I can tell all this behavior would be difficult to achieve with just tags, especially if we want to have more fine control over implanter draw lists in the future.